automated install/basic-install.sh

```diff
@@ -968,7 +979,7 @@ fi
 
 echo ":::"
 echo "::: The install log is located at: /etc/pihole/install.log"
-echo "::: View the web interface at http://pi.hole/admin or http://${IPv4addr%/*}/admin"
+echo "::: View the web interface at http://pi.hole/admin or http://${IPv4_address%/*}/admin"
 }
 
 main "$@"
```

Comment:
Could we get a dry-run that can prevent execution of this to allow just sourcing the functions and running them one by one?

This would allow unit tests to be written for the functions.  Though the test framework could also just comment this line out it may help with manual testing too, to not run everything at once.